server/services/AshesUserService.js

```diff
@@ -153,6 +154,28 @@ class UserService extends EventEmitter {
         });
     }
 
+    async updateElosFromGame(game) {
```

Comment:
Userservice shouldn't know about game. I think better to keep that knowledge in calling code.

Reply:
yeah, I wasn't sure about polluting gamerouter, but I didn't like it here either. Will change